dasgo/resources.go

```diff
@@ -956,71 +973,74 @@ const (
 // Guild Preview Object
 // https://discord.com/developers/docs/resources/guild#guild-preview-object-guild-preview-structure
 type GuildPreview struct {
-	ID                       string     `json:"id,omitempty"`
-	Name                     string     `json:"name,omitempty"`
-	Icon                     string     `json:"icon,omitempty"`
-	Splash                   string     `json:"splash,omitempty"`
-	DiscoverySplash          string     `json:"discovery_splash,omitempty"`
-	Emojis                   []*Emoji   `json:"emojis,omitempty"`
-	Features                 []*string  `json:"features,omitempty"`
-	ApproximateMemberCount   int        `json:"approximate_member_count,omitempty"`
-	ApproximatePresenceCount int        `json:"approximate_presence_count,omitempty"`
-	Description              *string    `json:"description,omitempty"`
-	Stickers                 []*Sticker `json:"stickers,omitempty"`
+	ID                       string     `json:"id"`
+	Name                     string     `json:"name"`
+	Icon                     string     `json:"icon"`
+	Splash                   string     `json:"splash"`
+	DiscoverySplash          string     `json:"discovery_splash"`
+	Emojis                   []*Emoji   `json:"emojis"`
+	Features                 []*string  `json:"features"`
+	ApproximateMemberCount   int        `json:"approximate_member_count"`
+	ApproximatePresenceCount int        `json:"approximate_presence_count"`
+	Description              *string    `json:"description"`
+	Stickers                 []*Sticker `json:"stickers"`
 }
 
 // Guild Widget Settings Object
 // https://discord.com/developers/docs/resources/guild#guild-widget-settings-object
 type GuildWidgetSettings struct {
-	Enabled   bool      `json:"enabled,omitempty"`
-	ChannelID Snowflake `json:"channel_id,omitempty"`
+	Enabled   bool      `json:"enabled"`
+	ChannelID Snowflake `json:"channel_id"`
 }
 
 // Guild Widget Object
 // https://discord.com/developers/docs/resources/guild#et-gguild-widget-object-get-guild-widget-structure*
 type GuildWidget struct {
-	ID            Snowflake  `json:"id,omitempty"`
-	Name          string     `json:"name,omitempty"`
-	InstantInvite string     `json:"instant_invite,omitempty"`
-	Channels      []*Channel `json:"channels,omitempty"`
-	Members       []*User    `json:"members,omitempty"`
-	PresenceCount int        `json:"presence_count,omitempty"`
+	ID            Snowflake  `json:"id"`
+	Name          string     `json:"name"`
+	InstantInvite *string     `json:"instant_invite"`
+	Channels      []*Channel `json:"channels"`
+	Members       []*User    `json:"members"`
+	PresenceCount int        `json:"presence_count"`
 }
 
 // Guild Member Object
+
+//GuildID not in api documentation
```

Comment:
We likely need to update with the new commits since the last Update PR but for now just remove this comment.